uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -141,22 +141,33 @@ <h4 class="modal-title">New Document</h4>
                 </div>
               </div>
             </div>
+
+            <!-- Display only for manager role and hidden removed from DOM for other uasers -->
+            <div class="col-xs-6" ng-if="isRole('manager')">
```

Comment:
but is there consistency here?

for example you have
```html
<div class="tab-pane" id="assignmentsTab" ng-hide="isRole('staff') || isRole('manager')">
```
why ng-hide in this case?